basic_nodes/src/main/java/club/mondaylunch/gatos/basicnodes/NumberOrderingNodeType.java

```diff
@@ -0,0 +1,65 @@
+package club.mondaylunch.gatos.basicnodes;
+
+import java.util.Map;
+import java.util.Set;
+import java.util.UUID;
+import java.util.concurrent.CompletableFuture;
+
+import club.mondaylunch.gatos.core.data.DataBox;
+import club.mondaylunch.gatos.core.data.DataType;
+import club.mondaylunch.gatos.core.graph.connector.NodeConnector;
+import club.mondaylunch.gatos.core.graph.type.NodeType;
+
+public class NumberOrderingNodeType extends NodeType.Process {
```

Comment:
maybe rename this number comparison? number ordering made me think of list sorting or smth at first.

Reply:
blame paul :P